### ISSUE TITLE
Fresh terraform deployment failed due to cmd/server failed to load locales

### ISSUE THREAD

**Issue by chaodaiG** [google/exposure-notifications-verification-server#1119]
### TL;DR

It was caught in terraform smoke test that terraform deploy failed on `cmd/server` service not up.

Error from stackdriver:
```
jsonPayload: {
timestamp: "2020-11-17T17:12:50.735224237Z"
build_id: "2535cb83-e820-4bcb-a16d-f80c775edc77"
caller: "server/main.go:54"
stacktrace: "main.main
	github.com/google/exposure-notifications-verification-server/cmd/server/main.go:54
runtime.main
	runtime/proc.go:204"
build_tag: "initial"
message: "failed to setup routes: failed to setup i18n: failed to load locales: open github.com/google/exposure-notifications-verification-server/internal/i18n/locales: no such file or directory"
}
```

Stackdriver link: https://pantheon.corp.google.com/logs/query;query=resource.type%3D%22cloud_run_revision%22%0Aresource.labels.service_name%3D%22server%22%0Aresource.labels.revision_name%3D%22server-l6sfs%22;timeRange=2020-11-17T17:11:05.934Z%2F2020-11-17T19:11:05.934346Z?pli=1&rapt=AEjHL4NPpyxZ6wSIdX5XLoHg3UL5aLF-pVfXcEvJLX6gEYcKMHA9K204ae6ZNsj_F3Zkh-TyhT1e72slgzGtAY9mbtOwZ7gUkw&project=apollo-boskos-ver-terraform-01

**Expected behavior**

Terraform deploy succeeds.

**Observed behavior**

### Reproduction

1.  Went to https://prow.k8s.io/view/gcs/oss-prow/logs/ci-en-verification-server-terraform-smoke/1328744881270558720

**Comment by sethvargo** [comment]
https://github.com/google/exposure-notifications-verification-server/pull/1122
